docs/vgclog.md

```diff
@@ -0,0 +1,423 @@
+
+The output of `-verbose:gc` is printed to STDERR by default. To print the log output to a file, append the `-Xverbosegclog` option. You can print the verbose GC log file to a succession of files, where each file logs a specified number of GC cycles, by using additional parameters of `-Xverbosegclog` option. For more information, see the command-line option [-Xverbosegclog](xverbosegclog.md).
+
+## Verbose GC log contents 
+
+The Verbose garbage collection logs are printed in XML format and consist of the following sections:  
+
+- a summary of your GC configuration, which is captured in the stanza beginning with the [`<INITIATIZED>` tag](vgclog.md#xml-tags-an-attributes) 
+
+- stanzas that contain information about the different garbage collection cycles run, including the GC increments and GC operations that made up the GC cycle. 
+
+The verbose GC logs are event-based, recording each *stop-the-world* event as it happens. While *stop-the-world* cycles are fully logged, some aspects of concurrent cycles are also logged, even though operations that are running concurrently with application threads are not logged. Concurrent operations run as part of a concurrent cycle, and these concurrent cycles begin and finish with *stop-the-world* events - events that are logged. (For some garbage collectors, the concurrent cycle also includes intermediate *stop-the-world* events). In addition, for some types of concurrent collections, verbose GC logs also include information about when certain targets are reached. 
+
+The highest level stanzas in the verbose GC log represent GC cycles and begin with xml tags that define the kickoff event and reason. Except for some of the more complex GC algorithms (such as the global marking phase GC of the balanced policy), a GC cycle, when complete, generally results in reclaimed memory for reuse. Each cycle consists of 1 or more GC increments and GC operations.  
+
+Events that make up a garbage collector cycle are listed in the stanza in a linear fashion, but are ‘nested’ by a start and end tag that identify the start and end of an event. Details of the event are recorded in the XML attributes associated with the tag and also within tags that are nested within the GC event's start and end tags.
+
+The following table shows the tagging that is used for the nested GC processes: 
+
+| GC Process   | Start and end tag in verbose GC log  | Details                                                         |
+|--------------|-------------------------------|-----------------------------------------------------------------|
+| GC cycle     | `<cycle-start>`, `<cycle-end>`| Completion of GC cycle results in reclaimed memory in most cases|
+| GC increment | `<gc-start>`, `<gc-end>`      | 1 or more GC increments make up a GC cycle                      |
+| GC operation | `<gc-op>`                     | 1 or more GC operations make up a GC increment. Examples of a GC operation include mark, sweep, scavenge.|
```

Comment:
'1 or more GC operations make up a GC increment' is not completely correct, since GC operations can also consist of several increments. It's more correct to say that '1 or more GC operations also make up a GC cycle'

I commented about this another document I'll copy paste it here too:

Gc cycles may contain 1 or more GC operations . Each GC operation typically consists of 3 phases (initial, main and final), although in a simple case all phases are reported together (may appear as a single phase). Phases may be performed as 1 increment but in some cases a phase (typically the main one) can be performed as a series of increments. 

Each increment can be a STW or a concurrent increment, or even hybrid of the 2.

Several GC operations or adjacent phases of different GC operations may be grouped and performed under a single STW increment.

Increments could be seen as physical division of a cycle, to accomplish a specific pace of garbage collection or to force a phase to be performed in STW or concurrent fashion.

GC operations could be seen as logic division of cycle, where each of them has specific functional purpose (or goal) to accomplish

.